### PR TITLE
Add information about the int08h server to ecosystem.md

### PR DESCRIPTION
**Pull request by int08h**
Provide information about `roughtime.int08h.com:2002`. Added in alphabetical order as per instructions in the `README`.

### CODE REVIEW

File: ecosystem.md [hunk 1]
```diff
@@ -5,6 +5,7 @@ servers. This file contains a brief description of how each server is
 provisioned. Refer to `README.md` for information about adding your server to
 the list.
 
+
```
Comment (cjpatton) [inline ecosystem.md]: Remove this newline.

File: ecosystem.md [hunk 2]
```diff
@@ -48,3 +49,24 @@ server](https://roughtime.googlesource.com/roughtime/#current-state-of-the-proje
...
+[leapsecond smeared NTP](https://developers.google.com/time/smear) servers. 
+Available at `roughtime.int08h.com:2002` its public key is stable and the service 
+is available 24/7. 
+
+The public key is available from the `README.md` in this project, 
+a [blog post at int08h](https://int08h.com/post/public-roughtime-server/), 
+and from a DNS `TXT` record: 
+
+```
+$ dig -t txt roughtime.int08h.com
```
Comment (cjpatton) [inline ecosystem.md]: Another time server (Chainpoint) points out how to `dig` their public key, so this line is a bit redundant. Please remove it.

File: ecosystem.md (hunk 1 above)
Comment (int08h) [reply ecosystem.md]: Removed

File: ecosystem.md (hunk 2 above)
Comment (int08h) [reply ecosystem.md]: Changed to reference the Chainpoint entry for instructions.

**Comment by int08h** [comment]
Requested edits made.

**Pull request merged**